nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -243,6 +243,7 @@ BtNavigator::navigateToPose()
     case nav2_behavior_tree::BtStatus::CANCELED:
       RCLCPP_INFO(get_logger(), "Navigation canceled");
       action_server_->terminate_all();
+      bt_->haltAllActions(tree_.root_node);
```

Comment:
`FAILED`, `SUCCEEDED` cases also needs to halt as terminal conditions.

Reply:
I will moving up before the run(), then